tests/models/independent/test_Project.py

```diff
@@ -693,7 +693,9 @@ def test_data_sources_derivation(self, proj, sample_index):
         # We don't care about that here, or about duplicates.
         expected = set(DERIVED_COLNAMES)
         observed = set(merged_columns)
-        assert expected == observed
+        # Observed may include additional things (like auto-added subsample_name)
+        for val in expected:
```

Comment:
Could be helpful to diagnose all failures in one shot with something like `assert set() == expected - observed`

Reply:
true. I guess I find the iterative way a little more readable though